### ISSUE TITLE
Naming conventions

### ISSUE THREAD

**Issue by oyron** [equinor/api-strategy#11]
OMNIA teams have asked for naming conventions for some services in Azure, e.g. API Management API names, API Management Product Names, etc.

**Comment by jokva** [comment]
Currently blocked by wishlist from business units.

**Comment by oyron** [comment]
I have sent this to Arild Eikeland in OMNIA API team for review. Will follow up with him

**Comment by oyron** [comment]
Completed. Revisions handled as separate issue.
